README.md

```diff
@@ -86,25 +86,25 @@ With all dependencies installed, all specs and features should pass:
 
 ### Running Appraisal suite
 
-In order to test against multiple versions of key dependencies, the [Appraisal](https://github.com/thoughtbot/appraisal) is used to generate multiple gemfiles, stored in the `gemfiles/` directory. Normally these will only run on Travis; however, if you want to run the full test suite against all gemfiles, run the following commands:
+In order to test against multiple versions of key dependencies, the [Appraisal](https://github.com/thoughtbot/appraisal) gem is used to generate multiple gemfiles.
+
+These are stored in the `gemfiles/` directory. Normally these will only run on Travis; however, if you want to run the full test suite against all gemfiles, run the following command:
 
-    [bundle exec] appraisal install
```

Comment:
I think this is needed to make sure all gems are actually installed at the start?

Reply:
When I was trying to fix things, this re-generates them. It's not like `bundle install` that doesn't regenerate if one is there. Which then causes rubocop / styling issues.

The gemfiles are currently under source control, so in theory you would only need to do the run command (Which would then bundle the specific gemfile).

I admit I'm a bit hazy on this, so I "could" be wrong. But I am fairly certain we shouldn't be advising anyone to run the install command as it changes all the sub-gemfiles

Reply:
Hm .. if I don't run `appraisal install`, but just `appraisal rake test`, I get "Could not find i18n-0.9.5 in any of the sources", but if I do run `appraisal install`, I get an error because my lockfiles are out of date. :thinking: 

I'm going to read up on this a bit.

Reply:
1. I've asked a question about cleaning up the lockfiles: https://github.com/thoughtbot/appraisal/issues/153
2. I think we need to keep `appraisal install` because we need gems installed. Since we warn that this is for maintainers, the resulting changes to the gemfiles are not that much of a problem, I think.
3. We may need to exclude the gemfiles from rubocop since we'll be re-generating them at some point and they'll use double-quoted strings and such.
4. (Not really related to this PR) I think the `[bundle exec]` bits here can go because the whole point is that appraisal will activate different bundles while running.

Reply:
There "ought" to be a command that uses the existing rules though and not re-installing them (like bundle does with bundle or bundle install).

I'll remove the files from rubocop tracking; what do you advise for the remainder. Should we wait to hear back from thoughtbot. As I'm not that knowledgable about Appraisals, I kind of got it working locally, but I didn't need to run the `install` command, but I may be doing something else unknowingly.

TL;DR - I don't really know what to do here.